lib/Minz/Extension.php

```diff
@@ -75,10 +69,8 @@ public function uninstall() {
 	/**
 	 * Call at the initialization of the extension (i.e. when the extension is
 	 * enabled by the extension manager).
-	 *
-	 * It must be redefined by child classes.
 	 */
-	public function init() {}
+	abstract public function init();
```

Comment:
NB Assuming that an extension would be useless if it didn't define this, anyway. ;-)